client/keys/add.go

```diff
@@ -95,12 +95,13 @@ func runAddCmdPrepare(cmd *cobra.Command, args []string) error {
 
 /*
 input
-	- bip39 mnemonic
-	- bip39 passphrase
-	- bip44 path
-	- local encryption password
+  - bip39 mnemonic
+  - bip39 passphrase
+  - bip44 path
+  - local encryption password
```

Comment:
Is this indent lint done by yourself or intellij?
Cosmos-sdk is same as former one.

Reply:
> This PR does not seem to implement second recommended check `Check if the Multisig contains the same name that will be overridden, and return an error to remind the users of trying a different name.`.

I didn't create that logic because there is a common flow to check whether to proceed with the override. So the user can decide to override based on his choice.

Reply:
> Is this indent lint done by yourself or intellij? Cosmos-sdk is same as former one.

done by intellij default formatter

Reply:
> done by intellij default formatter
When I use the commit feature provided by intellij or goland, lint automatically runs and changes are made in unwanted areas, so I am doing git commit in the terminal without using it. 
As in this comment, if lint runs and make changes in unwanted area, is it better to revert it?

> I didn't create that logic because there is a common flow to check whether to proceed with the override. So the user can decide to override based on his choice.

I thought this recommendation is for the case below.
There is normal keys named `account7, account8` and user want to make new multisig key named `account8` from keys `account7, account8`. In this case, even though user confirm this process, process will be fail as below img.
 
<img width="557" alt="스크린샷 2022-10-19 오후 5 07 16" src="https://user-images.githubusercontent.com/55660267/196633559-e5b8468f-0d54-4b8b-b85d-489e95406b40.png">

In upper case, `verifyMultisigTarget`  will return no error and normal key `account8` will be deleted.  But line 160, normal key `account8` is called again which leads an error `key not found`.

So I think that why audit recommends `Check if the Multisig contains the same name that will be overridden, and return an error to remind the users of trying a different name` to prevent creating a multisig key with the same name as the normal key to be used for making multisig key.

Reply:
Oh you already added logic to cover my error case at line 350 😄 .